src/methods/create.js

```diff
@@ -726,7 +727,7 @@ export class CreateImplementation {
       let next = Get(realm, obj, indexName);
 
       // c. If Type(next) is not an element of elementTypes, throw a TypeError exception.
-      if (elementTypes.indexOf(Type(realm, next)) < 0) {
+      if (elementTypes.indexOf(Type(realm, next)) < 0 || (realm.isInPureScope() && next instanceof AbstractValue)) {
```

Comment:
We shouldn't ignore things that might be errors just because we're in pure scope. Pure scope doesn't matter here.

The cases that you care about is the cases where the list of elementTypes is the default `["Undefined", "Null", "Boolean", "String", "Symbol", "Number", "Object"]`.

That list has all types on it so it doesn't matter what type the abstract value is.

If the elementTypes list is the default, you can just skip this check.

Reply:
Would you mind making this change to my PR? I’m without my Mac on PTO :)